pkg/apis/extensions/types.go

```diff
@@ -953,6 +955,15 @@ type RunAsUserStrategyOptions struct {
 	Ranges []UserIDRange
 }
 
+// RunAsGroupStrategyOptions defines the strategy type and any options used to create the strategy.
+type RunAsGroupStrategyOptions struct {
```

Comment:
Can we rename this to `GroupStrategyOptions` and share it with SupplementalGroups & FSGroup?

Reply:
wouldnt that be backward incompatible ?

Reply:
My understanding is that the go interfaces are allowed to change, as long as the json interface does not. In this case it would have no affect on the serialized json, so I think it's ok (but not positive).

Reply:
@tallclair is right re: backward compat.

Reply:
still outstanding?

Reply:
i plan to address this in later  PR